projects/components/src/time-range/time-range.component.ts

```diff
@@ -108,6 +108,8 @@ export class TimeRangeComponent {
   }
 
   public setToFixedTimeRange(timeRange: FixedTimeRange): void {
+    timeRange.startTime.setSeconds(0, 0);
```

Comment:
Let's add a comment here for future reference, in case someone apart from us both reviews this code.

Reply:
✅ Added comment